dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.rest.client-3.1.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.org.eclipse.microprofile.rest.client-3.1
+singleton=true
+# io.openliberty.mpCompatible-x.x comes from io.openliberty.microprofile.cdi.api features
+-features=\
+  io.openliberty.jakarta.annotation-2.1; ibm.tolerates:="3.0", \
+  io.openliberty.microprofile.cdi.api-4.0; ibm.tolerates:="4.1", \
```

Comment:
Similar here.  cdi.api-4.0 includes 6.0 and 6.1 toleration which doesn't seem correct.

Reply:
as above, io.openliberty.microprofile.cdi.api-4.0 had mpCompatible-7.0 toleration added